db/migrate/20210614145845_add_email_name_id_format_to_service_providers.rb

```diff
@@ -0,0 +1,17 @@
+class AddEmailNameIdFormatToServiceProviders < ActiveRecord::Migration[6.1]
+  def change
+    add_column :service_providers, :allow_email_nameid_format, :boolean, default: false
+
+    reversible do |dir|
+      dir.up do
```

Comment:
Note that we will also update the config to include this attribute as well but we can't merge that until this is deployed since our ServiceProvider seeder will try to set all of the attributes passed 😕. Therefore this will ensure that in the meantime all SPs have the correct setting.

Reply:
you can update the SP seeder to skip an attribute?

Reply:
You mean as a separate PR? I think that's due for a larger refactor and this seems like it should work fine for this issue. In my mind the issue is that the currently deployed code will break if we add an attribute to the config, so we're stuck until our next version regardless 😞. That said, I might not fully understand how to navigate implementing these kind of changes so if there's a better way to handle this please let me know!

Reply:
Well any time we have to sync changes between a deploy and the config repo... is gonna be a problem because we have 4+ environments all recycling at different times

Sometimes the easiest thing is to put out a change ahead of time that can ignore fields so we can not worry about breaking things

Reply:
Right - that's why my plan here was to have redundant configurations when this PR is deployed (b/c both the DB and the config will match) and then to fully switch over after the previous version is no longer running.

I'll add a ticket to refactor the SP seeder but unless you think that needs to block this change we're stuck until the next deploy in either case.